frontend/src/scenes/sessions/SessionsPlayer.tsx

```diff
@@ -7,7 +7,7 @@ export default function SessionsPlayer({ events }: { events: eventWithTime[] }):
     const target = useRef<HTMLDivElement | null>(null)
 
     useEffect(() => {
-        if (target.current) {
+        if (target.current && events) {
```

Comment:
@macobo the only thing I'm not 100% convinced of is this `&& events` part. Without it the back/forward buttons would crash, as rrwebPlayer got initialized with some empty data when going "forward" from /sessions to /sessions?id=..

Seeing how this `events` is not in the `[]` deps array I'm not sure if what I'm doing is legit or not.